src/main/java/org/jabref/logic/importer/fileformat/PdfImporter.java

```diff
@@ -0,0 +1,9 @@
+package org.jabref.logic.importer.fileformat;
+
+import org.jabref.logic.importer.Importer;
+
+/**
+ * Intermediate class to bundle all PdfImporters
+ */
+public abstract class PdfImporter extends Importer {
```

Comment:
I personally see no value on introducing this.

Reply:
It helps persons navigating using class hierarchies instead of naming conventions.  The former is supported by Java tooling, the latter needs to be documented etc